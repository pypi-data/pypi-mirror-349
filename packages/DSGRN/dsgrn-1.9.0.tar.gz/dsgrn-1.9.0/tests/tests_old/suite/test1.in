#!/bin/bash
$1 network ../networks/3D_Cycle_H.txt morsegraph graphviz '[["X1",[3,3,"FF9E00"],[0,1,2]],["X2",[1,2,"4"],[0,1]],["X3",[2,1,"8"],[0]]]'
