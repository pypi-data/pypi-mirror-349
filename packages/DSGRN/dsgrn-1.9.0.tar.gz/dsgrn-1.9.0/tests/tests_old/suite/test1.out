digraph {
0[label="PC {X1, X3}"];
1[label="FP { 3, 1, 1 }"];
0 -> 1;
}
